src/js/tracker.js

```diff
@@ -1112,21 +1095,20 @@
 		function getOptimizelyXSummary() {
 			var state = getOptimizelyXData('state');
 			var experiment_ids = state.getActiveExperimentIds();
-			var experiments = getOptimizelyXData('data', 'experiments');
 			var visitor = getOptimizelyXData('visitor');
 
 			return map(experiment_ids, function(activeExperiment) {
 				var variation = state.getVariationMap()[activeExperiment];
-				var variationName = variation.name;
-				var variationId = variation.id;
-				var visitorId = visitor.visitorId;
+				var variationName = (variation && variation.name && variation.name.toString()) || null;
+				var variationId = variation && variation.id;
+				var visitorId = (visitor && visitor.visitorId && visitor.visitorId.toString()) || null;
 				return {
-					experimentId: parseInt(activeExperiment),
+					experimentId: parseInt(activeExperiment) || null,
```

Comment:
Is zero a legitmate value for activeExperiment?  Because here you would replace it with null.  Same for variationId.

Reply:
I don't believe so. In all my testing Experiements seem to be around 9 or 10 digit numbers.